src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -329,6 +330,23 @@ private void RunAssembly(string assemblyPath, TestFilter filter)
             }
         }
 
+
+        private void GenerateTestOutput(XmlNode testResults, string assemblypath)
+        {
+            if (!Settings.UseTestOutputXml)
+                return;
+#if NETCOREAPP1_0
+#else
+            var path = Path.Combine(Settings.TestOutputXml,$"{Path.GetFileNameWithoutExtension(assemblypath)}.xml");
```

Comment:
Space usually after comma.
Also, `Path.ChangeExtension` is an alternative to string templating.

Reply:
Yes, that's better